corehq/apps/app_manager/decorators.py

```diff
@@ -51,4 +51,4 @@ def _no_conflict(*args, **kwargs):
     return _no_conflict
 
 require_can_edit_apps = require_permission(Permissions.edit_apps)
-require_deploy_apps = login_and_domain_required  # todo: can fix this when it is better supported
+require_deploy_apps = require_can_edit_apps
```

Comment:
I think the "right" behavior would be to make a new permission for being able to deploy apps that encompasses some of this, but not sure how hard that is. our permissions framework is not all that extensible which is why it's so limited.